com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/parser/UriParametersParser.java

```diff
@@ -133,6 +133,8 @@ private void addHostBasedUris(Set<String> uris, String domain, List<String> host
             .stream()
             .map(routesMap -> (String) routesMap.get(SupportedParameters.ROUTE))
             .filter(Objects::nonNull)
+            .map(ApplicationURI::new)
+            .map(ApplicationURI::toString)
```

Comment:
Perhaps, instead of invoking new and toSting, we can just call a static version of the stripScheme method, since this is essentially the only thing we're using the ApplicationURI object for.